### PR TITLE
Feature/pxblue 2815 r22 release notes

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # .

#### Changes proposed in this Pull Request:
- R22 release notes
- 
- 

#### To Test:
-

### CODE REVIEW

**Comment by huayunh** [comment]
will there be a package update section? Like, "besides the change of package handles from @pxblue to @brightlayer-ui, there are minor updates to the following packages too."

**Comment by jeffvg** [comment]
> will there be a package update section? Like, "besides the change of package handles from @pxblue to @brightlayer-ui, there are minor updates to the following packages too."

wasn't sure what to do with that but we have a section "Developer Migration Documentation" in release note pointing to migration doc

**Comment by huayunh** [comment]
Yes. So there are two different parts happening with the packages. There is the BLUI migration, and there are functional updates to a few packages.

See #505

**Comment by andrupu** [comment]
It looks like we are calling PXBlue / BLUI a "design system" in R22.mdx and a "platform" in Summary.mdx.  Should we call it one or the other in both, so there is no confusion?  Maybe word it like "The Design System of Power Xpert Blue (PXBlue) was rebranded to Brightlayer User Interface (Brightlayer UI)...??  Maybe "Design System Platform" to combine the two terms?

**Comment by andrupu** [comment]
It looks like we are calling PXBlue / BLUI a "design system" in R22.mdx and a "platform" in Summary.mdx.  Should we call it one or the other in both, so there is no confusion?  Maybe word it like "The Design System of Power Xpert Blue (PXBlue) was rebranded to Brightlayer User Interface (Brightlayer UI)...??  Maybe "Design System Platform" to combine the two terms?

**Comment by andrupu** [comment]
It looks like we are calling PXBlue / BLUI a "design system" in R22.mdx and a "platform" in Summary.mdx.  Should we call it one or the other in both, so there is no confusion?  Maybe word it like "The Design System of Power Xpert Blue (PXBlue) was rebranded to Brightlayer User Interface (Brightlayer UI)...??  Maybe "Design System Platform" to combine the two terms?

**Comment by daileytj** [comment]
Looks mostly good to me, don't forget to update this to add the version numbers. I don't think the little blurb under the package summary is necessary, but I don't think it hurts either so your call on that one.

**Comment by daileytj** [comment]
Screen Shot 2022-01-10 at 11 08 48 AM

**Comment by jeffvg** [comment]
> Screen Shot 2022-01-10 at 11 08 48 AM

added version info

File: src/docs/release-notes/R22/R22.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+-   Added React Empty State card variation
+
+### Icons
+
+-   Added 38 new icons
+-   Fixed an icon clipping issue that caused wrong rendering in icon fonts
+
+### Themes
+
+-   Minor updates to Angular themes and dark theme
```
Comment (huayunh) [inline src/docs/release-notes/R22/R22.mdx:23]: According to our change logs, we didn't publish any theme changes last quarter. May want to leave this out.

Comment (jeffvg) [reply src/docs/release-notes/R22/R22.mdx:23]: we did but small items

image

Comment (huayunh) [reply src/docs/release-notes/R22/R22.mdx:23]: Are they merged to master?

Comment (jeffvg) [reply src/docs/release-notes/R22/R22.mdx:23]: They are not in master, ok

**Review by huayunh — COMMENTED** [review]
@joebochill word smith?

File: src/docs/release-notes/R22/R22.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+-   Added Multi-Line Text Field component and Progress Icons
+
+### Miscellaneous
+
+-   Bug fixes & minor updates
+
+### Package Summary
+
+Besides the package migrations to the new @brightlayer-ui handler, we also made some minor updates to the following packages.
```
Comment (emclaug2) [inline src/docs/release-notes/R22/R22.mdx:37]: I agree with Tom, this blurb "Besides the package.... following packages" could be removed to match the formatting from last release.

File: src/docs/release-notes/R22/Summary.mdx [hunk 3]
```diff
@@ -0,0 +1 @@
+Rebranded the PX Blue platform to Brightlayer UI.
```
[the remark this answers is not in the document]
Comment (emclaug2) [reply src/docs/release-notes/R22/Summary.mdx:1]: "Rebranded the PX Blue platform to Brightlayer UI, including NPM resources, GitHub and all doc sites." ?

File: src/docs/release-notes/R22/R22.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+-   [Brightlayer UI](https://brightlayer-ui.github.io/) documentation site
+-   Github organization migrated to [Brightlayer UI](https://github.com/brightlayer-ui)
+-   Component libraries migrated to [Brightlayer UI](https://brightlayer-ui-components.github.io/)
+-   NPM packages migrated to [Brightlayer UI](https://www.npmjs.com/org/brightlayerui)
+
+### Component Libraries
+
+-   Updated Angular Score Card component styles
+-   Updated React Info List Item with conditional title and subTitle rendering
+-   Added React Empty State card variation
```
Comment (emclaug2) [inline src/docs/release-notes/R22/R22.mdx:14]: "Added React Empty State card variation"

Can you specify that this is for the storybook demo & not directly impacting the component library?

Comment (jeffvg) [reply src/docs/release-notes/R22/R22.mdx:14]: yes, guess this too can be removed

**Review by emclaug2 — APPROVED** [review]
I think you just need to run prettier, but I'm happy with the changes.

**Pull request merged**